com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,73 @@
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+build-options:
+    prepend-pkg-config-path: "/usr/lib/i386-linux-gnu/pkgconfig"
+    ldflags: "-L/app/lib32"
+    prepend-path: "/usr/lib/sdk/toolchain-i386/bin"
+    env:
+        CC: "i686-unknown-linux-gnu-gcc"
+        CXX: "i686-unknown-linux-gnu-g++"
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=dri"
+  - "--allow=multiarch"
+
+sdk-extensions:
+  - 'org.freedesktop.Sdk.Compat.i386'
+  - 'org.freedesktop.Sdk.Extension.toolchain-i386'
+
+add-extensions:
+    'org.freedesktop.Platform.Compat.i386':
+        directory: "lib/i386-linux-gnu"
+        version: '19.08'
+
+    'org.freedesktop.Platform.GL32':
+        directory: "lib/i386-linux-gnu/GL"
+        version: '1.4'
+        versions: '19.08;1.4'
+        subdirectories: true
```

Comment:
Add `add-ld-path: lib` somewhere here.

Reply:
When I do it says it "fails to load driver `i965`"

Reply:
Try copying the extension definition from Steam as is. Maybe `lib/dri` in `merge-dirs` is needed or something else.

Reply:
Nope. Still fails to load driver with the GL def from Steam as-is

Reply:
It's probably a separate issue. `add-ld-path: lib` is required for the extension to work anyway.

Reply:
> `add-ld-path: lib` is required for the extension to work anyway.
I honestly don't know about that. `LD_LIBRARY_PATH` is set properly without, and when using it and manually fixing `LD_LIBRARY_PATH` afterwards I get to the GLFW error

Reply:
And if the extension didn't work, it would say something like "invalid binary format" or ld.so's "no such file"

Reply:
You're mixing `.Compat.i386` extension with `.GL32` extension. Without the later, you'll be able to run 32-bit binaries, but not able to use 32-bit opengl. And `No GLXFBConfigs returned` indicates exactly this case - 32bit binary was run, but couldn't create opengl context.

Reply:
"Fixed" (worked around locally in testing) by setting `LIBGL_DRIVERS_PATH="/app/lib/i386-linux-gnu/GL/default/lib/dri"` (which it somehow isn't by default?

Reply:
Again, copy the extension from Steam. This is likely due to `lib/dri` missing from `merge-dirs` property.

Reply:
You mean this? That's already what I'm using

```yaml
    'org.freedesktop.Platform.GL32':
        directory: "lib/i386-linux-gnu/GL"
        version: '1.4'
        versions: 19.08;1.4
        subdirectories: true
        #no-autodownload: true
        autodelete: false
        add-ld-path: 'lib'
        merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
        download-if: active-gl-driver
        enable-if: active-gl-driver
```

Reply:
Then you're using something that is not part of this PR.

Reply:
For internal testing, yes.

Reply:
This is the correct way to add the GL32 extension, so probably yes.